scripts/test-truncating-of-client-hello.py

```diff
@@ -188,7 +188,7 @@ def run_all(conversations):
         try:
             runner.run()
         except:
-            # TBD: print(traceback.format_exc())
+            print(traceback.format_exc())
```

Comment:
You are right. Printing should be after every test run with print of traceback on relative position.